develop/tutorials/articles/110-portlets/15-javascript/04-npm/08-npm-sdk.markdown

```diff
@@ -1,12 +1,12 @@
 # Liferay npm SDK Assistant [](id=liferay-npm-sdk-assistant)
 
-@product@ has several components that rely on npm build tools. *npm SDK* is an 
-abstract umbrella term that refers to all these components as a whole. It can be 
-hard keeping track of all the different component versions and which ones are 
-required to use certain features. The Liferay npm SDK Assistant is a CLI tool 
-that provides a list of features and their supported component versions, along 
-with your current component versions, so you can update your components to a 
-version that supports the features you need. 
+@product@ has several components--collectively known as *npm SDK*--on which
```

Comment:
@nealrobbins I think it's important that we mention that the components rely on npm. Then we can mention the npm sdk afterwards. Also note that this tool is still being tweaked, so these docs may change slightly.

Reply:
The reason we need to mention is separately at first is because npm is separate from Liferay. Liferay just happens to use the tool. The npm SDK is our own term for those Liferay-specific components that rely on npm. So, we have to be very clear with the wording.

Reply:
Why is important to mention that the components rely on npm? What bad thing might happen if we don't? If I understand correctly--which is admittedly unlikely--this article has is about the relationship between features and the components they rely on--not on the relationship between the components and npm. The dependency on npm seems irrelevant to the rest of the article.

I'm not trying to pick a fight (though I like a good brawl as much as the next man). I just don't want to say "sure, whatever you say" without understanding why.

Reply:
Wait. Rereading what you said...do we just want to make clear that this article is dealing only with npm-dependent components and not any others?

Reply:
Yeah. The npm SDK Assistant tool is only for npm-dependent components. This tool just helps you keep track of the component versions and available features so you can determine what features you should be able to use.

Reply:
Haha. No brawling required. You're right, this article is about the relationship between features and the components they rely on. We just want to make it clear that these are dealing only with Liferay components that use npm and not any others, as you said :-)

Reply:
Ok, I can take care of that.